testing/test_full_pipeline_dump.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+
+from neuraxle.base import Identity, ExecutionContext, JoblibStepSaver
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.misc import FitTransformCallbackStep, TapeCallbackFunction
+from neuraxle.steps.output_handlers import OutputTransformerWrapper
+
+PIPELINE_NAME = 'saved_pipeline'
+
+DATA_INPUTS = np.array(range(10))
+EXPECTED_OUTPUTS = np.array(range(10, 20))
+
+
+def test_integrated_should_save_and_load_full_pipeline_dump(tmpdir):
+    # Given
+    tape_fit_callback_function = TapeCallbackFunction()
+    tape_transform_callback_function = TapeCallbackFunction()
+    pipeline = Pipeline([
+        ('step_a', Identity()),
+        ('step_b', OutputTransformerWrapper(
+            FitTransformCallbackStep(tape_fit_callback_function, tape_transform_callback_function)
+        ))
+    ], cache_folder=tmpdir).set_name(PIPELINE_NAME)
+
+    # When
+    pipeline, outputs = pipeline.fit_transform(DATA_INPUTS, EXPECTED_OUTPUTS)
+    pipeline.save(ExecutionContext(tmpdir), full_dump=True)
+
+    # Then
+    loaded_pipeline = Identity().set_name(PIPELINE_NAME).load(ExecutionContext(tmpdir), full_dump=True)
```

Comment:
I should Instead use a LoadableStep(PIPELINE_NAME).load(ExecutionContext(tmpdir), full_dump=True), and not save the savers (that is retarded)

Reply:
done way cleaner will open in a new pr instead